app/services/login/after_login_actions.rb

```diff
@@ -37,9 +35,7 @@ def login_type
 
     def send_reactivation_email?
       availability = AcceptableVerifiedCredentialAdoptionService.new(@current_user).perform
-      if availability[:reactivation_email]
-        VANotifyReactivationEmailJob.perform_async
-      end
+      VANotifyReactivationEmailJob.perform_async if availability[:reactivation_email]
```

Comment:
This PR removed this particular job, should this still be included?

Reply:
It will be included in a future PR. Trying to keep things small and focused. But thanks for double checking!